### DISCUSSION TITLE
Feature request: Add date picker

### DISCUSSION THREAD

**Discussion by ssakib4040** [saadeghi/daisyui#1558] [Ideas / feature requests]
Hi, 

Is it possible to add a date picker?
I am looking for something like this: 
![image](https://user-images.githubusercontent.com/101496701/214630639-a7cdaec0-cec4-46b4-918b-f18f094640b0.png)

**Comment by saadeghi** [comment]
Unfortunately it's not possible to apply styles to the native HTML datepicker input. Any JS datepicker plugin you see on the internet uses divs and buttons to visually show a datepicker and then uses JS to make it functional. That is not possible using CSS only so you may want to use a JS plugin for that.
